src/dbcbs_utils.hpp

```diff
@@ -89,6 +89,32 @@ typedef boost::heap::d_ary_heap<
       boost::heap::compare<compareFocalHeuristic>, boost::heap::mutable_<true>>
       focalset_t;
 
+// for cbs-style optimization
+struct HighLevelNodeOptimization {
+    // std::vector<dynobench::Trajectory> trajectories;
+    MultiRobotTrajectory multirobot_trajectory;
+    std::unordered_set<size_t> cluster; // robot idx for the joint optimization
```

Comment:
This should be a vector of sets (*all* clusters, not just one; the root node has N clusters of cardinality 1)